trigger-jenkins-pipeline/trigger-jenkins-pipeline.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: trigger-jenkins-pipeline
```

Comment:
`trigger-jenkins-job` might be a more suitable name as not all Jenkins jobs are pipelines.